cypress/cypress.config.js

```diff
@@ -5,6 +5,8 @@ export default defineConfig({
   e2e: {
     saucelabs: {
       buildName: 'SauceDemo - Cypress',
+      branch: 'main',
+      project: 'CY examples',
```

Comment:
Alternative to above: use repo name.

```suggestion
      buildName: 'Cypress job on git sha abcdef',
      branch: 'main',
      project: 'saucelabs/visual-examples',
```

Reply:
I did it the same way as wdio. I recommend to create a ticket for renaming and unification for all client bindings.

Reply:
@ebekebe I think we should proceed with this mr.